### ISSUE TITLE
Code issues

### ISSUE THREAD

**Issue by wx24598** [ChenhongyiYang/QueryDet-PyTorch#50]
Hello author, your job is very good. Can you answer the meaning of the following code for me?

        for i in range(-1*self.context, self.context+1):
            for j in range(-1*self.context, self.context+1):
                sparse_ys.append(ys+i)
                sparse_xs.append(xs+j)

        sparse_ys = torch.cat(sparse_ys, dim=0)
        sparse_xs = torch.cat(sparse_xs, dim=0)

        good_idx = (sparse_ys >= 0) & (sparse_ys < fh) & (sparse_xs >= 0)  & (sparse_xs < fw)

**Comment by ChenhongyiYang** [comment]
Hi, the code is used to add context information around small objects, which is important to the performance. Thank you!

**Comment by wx24598** [comment]
非常感谢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ChenhongyiYang/QueryDet-PyTorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月17日(星期五) 凌晨2:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ChenhongyiYang/QueryDet-PyTorch] Code issues (Issue #50)





 
Hi, the code is used to add context information around small objects, which is important to the performance. Thank you!
